### DISCUSSION TITLE
Packages required

### DISCUSSION THREAD

**Discussion by TheAlexCH** [erigontech/erigon#3072] [Q&A] [answered]
While installing it on ubuntu 18.04 I got some errors. What package must be installed prior to "make erigon"?

**Comment by AskAlexSharov** [comment]
You need do “some actions” to fix “some errors”. 
Don’t want to be rude, but next time provide error message please (or read it - maybe there system say you what package required). 

Default gcc and default golang must be enough.

**Comment by TheAlexCH** [comment]
Requirement prior to installing:

GCC 10+
Go 1.16+

That was not clearly visible but I figured it out. Maybe add it to the requirements as those are not default installed on an AWS Ubuntu 18.04 instance which probably many would use to fire up a test

**Comment by AskAlexSharov** [comment, marked as answer]
Your PR’s are welcome:  https://github.com/ledgerwatch/erigon/blob/devel/README.md#system-requirements

**Reply by TheAlexCH** [reply]
Thanks a lot. Here I could also use an updated link as I would like to try accessing data directly. https://github.com/ledgerwatch/erigon/blob/devel/cmd/rpcdaemon/docs/programmers_guide/db_faq.md

This is from the deamon doc, link is broken.

**Reply by AskAlexSharov** [reply]
fixed
